sgkit/stats/aggregation.py

```diff
@@ -69,3 +72,59 @@ def count_alleles(ds: Dataset) -> DataArray:
     assert AC.shape == (n_variant, n_allele)
 
     return DataArray(data=AC, dims=("variants", "alleles"), name="variant_allele_count")
+
+
+def _swap(dim: Dimension) -> Dimension:
+    return "samples" if dim == "variants" else "variants"
+
+
+def call_rate(ds: Dataset, dim: Dimension) -> Dataset:
+    odim = _swap(dim)[:-1]
+    n_called = (~ds["call_genotype_mask"].any(dim="ploidy")).sum(dim=dim)
+    return xr.Dataset(
+        {f"{odim}_n_called": n_called, f"{odim}_call_rate": n_called / ds.dims[dim]}
+    )
+
+
+def genotype_count(ds: Dataset, dim: Dimension) -> Dataset:
```

Comment:
FYI this should be https://github.com/pystatgen/sgkit/issues/29#issuecomment-656691069 instead.

Reply:
This is now fixed.